frontend/src/pages/ErrorsV2/ErrorsV2.tsx

```diff
@@ -169,141 +246,182 @@ const ErrorsV2: React.FC<React.PropsWithChildren<{ integrated: boolean }>> = ({
 		},
 		[showLeftPanel],
 	)
+}
 
-	return (
-		<>
-			<Helmet>
-				<title>Errors</title>
-			</Helmet>
+type ErrorDisplayProps = {
+	errorGroup?: GetErrorGroupQuery['error_group']
+	integrated: boolean
+	isBlocked: boolean
+	isBlockedLoading: boolean
+	isErrorGroupError?: boolean
+	isErrorState: boolean
+	loading: boolean
+}
+function ErrorDisplay({
+	errorGroup,
+	integrated,
+	isBlocked,
+	isBlockedLoading,
+	isErrorGroupError,
+	isErrorState,
+	loading,
+}: ErrorDisplayProps) {
+	const { error_secure_id } = useParams<Params>()
 
-			<Box cssClass={styles.container} borderTop="dividerWeak">
-				<SearchPanel />
+	switch (true) {
```

Comment:
I've got a controversial habit of using `switch (true)` to un-nest ternaries. Testing the waters.

This feels acceptable because the switch/true is isolated in its own function. It's a poor man's JS [do expression](https://github.com/tc39/proposal-do-expressions)

Reply:
It looks like it didn't need to be a ternary in the beginning, but my new logic was a little more complicated.